go.mod

```diff
@@ -1,20 +1,34 @@
 module github.com/cosmos/gogoproto
 
-go 1.19
+go 1.21
```

Comment:
this warrants a changelog

Reply:
Now we don't import cosmossdk.io/x/tx, can we revert the bumping of the go.mdo (and ci)?